app/forms/collection_form.rb

```diff
@@ -3,91 +3,71 @@
 # Form for a Collection
 class CollectionForm < ApplicationForm
   accepts_nested_attributes_for :related_links, :contact_emails, :managers, :depositors, :reviewers
-  before_validation do
-    blank_managers = managers_attributes.select(&:empty?)
-    next if blank_managers.empty?
-
-    self.managers_attributes = managers_attributes - blank_managers
-  end
-  validates :managers_attributes, length: { minimum: 1, message: 'must have at least one manager' } # rubocop:disable Rails/I18nLocaleTexts
```

Comment:
I reorganized the form so it's a bit more readable to my eyes. I tried to group things with like things, i.e., attributes, validations, instance methods.

Reply:
Our pattern for forms has been to group an attribute with all things related to that attribute (validations, etc.). I'm not opposed to changing that pattern, but don't like changing it in just this form.

Reply:
This is also going to cause merge headaches with my PR that also touches this form.

Reply:
I'll revert the form organization changes and may or may not consider making them again in a different PR later